test/gst-msdk/decode/av1.py

```diff
@@ -23,7 +23,7 @@ def before(self):
   def test(self, case):
     vars(self).update(spec[case].copy())
 
-    dxmap = {".ivf" : "ivfparse", ".av1" : "ivfparse", ".webm" : "matroskademux"}
+    dxmap = {".ivf" : "ivfparse", ".av1" : "ivfparse ! av1parse ", ".webm" : "matroskademux"}
```

Comment:
`av1parse` should be added after all demuxers.  Thus, it would be better to just prepend before `msdkav1dec` in the gstdecoder setting on line 32.